src/server/action_handlers/test-action.js

```diff
@@ -16,7 +15,7 @@ export const instructions = () => (
 // either in environment variables or organization.features json data
 // Besides this returning true, "test-action" will also need to be added to
 // process.env.ACTION_HANDLERS
-export async function available(organizationId) {
+export async function available() {
```

Comment:
The purpose of this file is to expose the arguments sent -- it doesn't matter that it's not used.  please add back.